android/java/org/chromium/chrome/browser/crypto_wallet/activities/BuySendSwapActivity.java

```diff
@@ -1368,9 +1367,6 @@ public void updateBuySendSwapAsset(
                     token.symbol, getResources().getDisplayMetrics().density, assetText, this, true,
                     (float) 0.5);
         }
-        updateBalance(
-                mCustomAccountAdapter.getTitleAtPosition(mAccountSpinner.getSelectedItemPosition()),
-                false);
```

Comment:
are you sure it's safe to remove it as we call `updateBuySendSwapAsset` from `resetSwapFromToAssets` as well

Reply:
There are three calls to `resetSwapFromToAssets`, the one in `adjustControl` doesn't do anything because of the `if`, the one in `onItemSelected` has `updateBalanceMaybeSwap` after it, and the one in `finishNativeInitialization` has `updateBalanceMaybeSwap` before it.

Reply:
ok, sounds good